process/smartContract/processorV2/processV2.go

```diff
@@ -1925,7 +1925,11 @@ func (sc *scProcessor) printScDeployed(vmOutput *vmcommon.VMOutput, tx data.Tran
 }
 
 // taking money from sender, as VM might not have access to him because of state sharding
-func (sc *scProcessor) processSCPayment(tx data.TransactionHandler, acntSnd state.UserAccountHandler) error {
+func (sc *scProcessor) processSCPayment(
+	tx data.TransactionHandler,
+	acntSnd state.UserAccountHandler,
+	acntRcv state.UserAccountHandler,
```

Comment:
Sometimes we say `acntRcv`, and sometimes `acntDst` (tiny inconsistency).

Reply:
updated to acntDst